dbtestutil_test.go

```diff
@@ -80,11 +85,12 @@ type rSliceElementForTesting struct {
 	IsUndefined bool
 }
 
-func (s *DbZuite) DbState() ([]rWorksheet, []rValueForTesting, []rSliceElementForTesting) {
+func (s *DbZuite) DbState() ([]rWorksheet, []rValueForTesting, []rParent, []rSliceElementForTesting) {
```

Comment:
This return tuple is getting pretty big. What's the rule of thumb for Golang on when to convert to a struct?

Reply:
Definitely at the point where we should convert it. I want to do this in a separate PR that only deal with this, rather than increase the size of a PR that has logic in it.

Reply:
👍